Source/IntegrationTests/TestFiles/LitTests/LitTest/comp/CompileWithArguments.dfy

```diff
@@ -5,10 +5,10 @@
 // RUN: %run --no-verify --target:js "%s" -- Javascript 2 >> "%t"
 // RUN: %run --no-verify --target py "%s" Python 1 >> "%t"
 // RUN: %run --no-verify --target go "%s" "Go go" 1 >> "%t"
-// RUN: %build --no-verify --target:cs "%s" --output:%s.dll
-// RUN: dotnet %s.dll "dotnet" "howdy" >> "%t"
-// RUN: dotnet %s.dll "dotnet" "hello" >> "%t"
-// RUN: dotnet %s.dll "dotnet" "aloha" >> "%t"
+// RUN: %build --no-verify --target:cs "%s" --output:%S/Output/CompileWithArgument.dll
+// RUN: dotnet %S/Output/CompileWithArgument.dll "dotnet" "howdy" >> "%t"
+// RUN: dotnet %S/Output/CompileWithArgument.dll "dotnet" "hello" >> "%t"
+// RUN: dotnet %S/Output/CompileWithArgument.dll "dotnet" "aloha" >> "%t"
```

Comment:
Why?

Reply:
Err, I don't recall. Might have to do with collisions on the file system if not running `dotnet build` in a separate directory.